.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,3 +8,21 @@ To submit a PR please make sure to follow the next steps:
 - [ ] Describe your changes with as much detail as you can.
 - [ ] Link any issues this PR is related to.
 - [ ] Remove the text above.
+
+<!--
+
+The PR will be reviewed by an osquery committer.
+Here are some common things we look for:
+
+- The code is formatted correctly, considering using `make format_check`.
+- Common utilities within `./osquery/utils` are used where appropriate (avoid reinventions).
+- Modern C++11 structures and patterns are used where appropriate.
```

Comment:
Shouldn't we strive to use Modern C++ in general?
What throws me off is that written like that Modern C++ seems bad?

C++11 is 8 years old, we compile with C++14.
I'm not saying we should use every feature of C++11/14 blindly, but surely write Modern C++, not pre-C++11 code, not C (unless for optimization or around system APIs where's necessary, but that should be very limited).

Reply:
We should, but "where appropriate" handles interaction with OS/third-party libraries. There are reasonable instances where we cannot use modern structures. But I will defer to recommendations here - if you have a better suggestion on how to write this - we are saying the same thing: use modern C++ as much as possible!

Reply:
Then I suggest changing with "where possible" :).
I think that's a bit stronger and making the case for "impossible" it's a bit more difficult.